### DISCUSSION TITLE
CUYUNIVERSITY Semester 4 | Fetch Failed

### DISCUSSION THREAD

**Discussion by rammmpr** [deaafrizal/forum#16] [Troubleshooting Room]
ijin bang dea, ini saya kadang2 dapet error Fetch Failed, gara gara apa ya bang? mohon bantuannya 

![Screenshot from 2023-11-29 20-05-10](https://github.com/deaaprizal/forum/assets/113332544/0c1a559a-1731-4e32-b1dc-ba0ee97162d6)

[Screencast from 29-11-23 20:27:52.webm](https://github.com/deaaprizal/forum/assets/113332544/25c7e206-abf5-48a2-a017-f5f2853cbb8b)

**Comment by deaafrizal** [comment]
udah coba compare code sama code gue di repository cuyanimelist nya? terutama code pas nge fetch nya.

**Reply by rammmpr** [reply]
belum sih bang
